deepchem/models/tensorgraph/chemnet_layers.py

```diff
@@ -0,0 +1,487 @@
+"""
+Contains implementations of layers used in ChemCeption and Smiles2Vec models.
+"""
+
+from __future__ import division
+from __future__ import unicode_literals
+
+__author__ = "Vignesh Ram Somnath"
+__license__ = "MIT"
+
+import numpy as np
+import tensorflow as tf
+from tensorflow.keras.layers import Conv2D, Concatenate, ReLU, Add, MaxPool2D
+
+
+class Stem(tf.keras.layers.Layer):
```

Comment:
Could you add a docstring explaining what this class does?

Reply:
I was looking at the Inception-ResNet paper (https://arxiv.org/pdf/1602.07261.pdf) to get help on the documentation, and noticed that the Stem segment is structured much more differently, atleast from what I could gather from the ChemCeption paper (https://arxiv.org/pdf/1706.06689.pdf).

In the ChemCeption paper, they say the segment has a single 4 x 4 convolutional layer with stride 2, which is what I implemented. But the structure between the two architectures is so different that the purpose is unclear to me now. The only thing I can think of the Stem Layer performs some sort of image downsampling.

Reply:
Done for all the layers requested.